.github/workflows/workflow_rerunner.yml

```diff
@@ -73,7 +73,7 @@ jobs:
   rerunner_tests:
     name: Rerunner Tests
     if: ${{ github.event_name == 'pull_request' && github.repository_owner == 'openvinotoolkit' }}
-    runs-on: aks-linux-2-cores-8gb-stats
```

Comment:
Do we really want to use the same runners pool for statistic jobs?

Reply:
Yes. The split was done to avoid fighting for the same hardware resources, it's not the case with `aks-linux-small`